app/models/user.rb

```diff
@@ -3,8 +3,10 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates_presence_of :password
   validates_presence_of :first_name
+  validates_presence_of :last_name
   enum role: [:default, :admin]
   has_secure_password
+
+  validates :password, confirmation: { presence: true, case_sensitive: true }
```

Comment:
This is a way better way to check this confirmation and make sure the model actually looks like it's supposed to.

Reply:
thanks 👍